### ISSUE TITLE
integration no longer works with HA 2022.4.0

### ISSUE THREAD

**Issue by tescophil** [LaggAt/hacs-govee#67]
Version, from the manifest file says **0.2.2**  I pulled the latest version from git, and the bug report says to check const.py for the version number, but all that this file contains is this :
```
"""Constants for the Govee LED strips integration."""

DOMAIN = "govee"

CONF_DISABLE_ATTRIBUTE_UPDATES = "disable_attribute_updates"
CONF_OFFLINE_IS_OFF = "offline_is_off"
CONF_USE_ASSUMED_STATE = "use_assumed_state"

COLOR_TEMP_KELVIN_MIN = 2000
COLOR_TEMP_KELVIN_MAX = 9000
```
Nothing in the logs

The integration says it failed to start, deleted it and tried to re-add which produced this error dialogue:

![image](https://user-images.githubusercontent.com/59442445/162041483-ec529df3-cc99-48fa-964d-c8b442233dbf.png)

**Comment by TheGamersCubeNL** [comment]
Same here

**Comment by Siju21** [comment]
Same here as well

**Comment by Merril-r** [comment]
Mine is also not working after upgrade to latest version of HA

**Comment by Florian-Schmidt** [comment]
Same here ...

**Comment by langerma** [comment]
same here

**Comment by fakoua** [comment]
Same here

**Comment by tescophil** [comment]
The problem persists with the latest version on my installation

**Comment by Florian-Schmidt** [comment]
> The problem persists with the latest version on my installation

Updated both and restarted HA? Did you try to remove Govee from custom_components?

For me it looked like a complete new installation starting with HACS even custom_components was still existing in my case.

**Comment by tescophil** [comment]
I don't use HACS, it caused performance problems the last time I had it installed with constant updates and restarts. I only use three custom components which are trivial to update with a simple bash script. If you now have to install HACS to use the Govee integration, then that will be the end of the road for me.

**Comment by Florian-Schmidt** [comment]
Govee integration still listed in HA or not? 

For me it hasn't been visible in HACS and HA. I totally added the integration again and created a new API key in the official app.

HACS shouldn't be the issue because any folders in custom_components should be loaded by HA.

**Comment by langerma** [comment]
as for me it did not work... the govee integration still shows that it is not configurable throught the ui

**Comment by dac73** [comment]
Just started working again for me after update to `Home Assistant 2022.4.4`

**Comment by tescophil** [comment]
Updated to 2022.4.4 and still have the same issue...

**Comment by IIAIronWolf** [comment]
It hasn't worked for me for maybe the last 6/7 HA updates.

**Comment by ahmadnassri** [comment]
upgrading to HA v2022.4.6 solved this issue for me

**Comment by tescophil** [comment]
No, still does not work after the 2022.4.6 upgrade, same "This integration does not support configuration via the UI" error message when trying to add the integration.

**Comment by tescophil** [comment]
Installation via HACS (begrudgingly) wortks OK, but don't see why this would be any different from installing directly from the repo...

**Comment by IIAIronWolf** [comment]
I don't know that this should have been closed. My setup is already through HACS and it is still not working...

**Comment by tescophil** [comment]
@IIAIronWolf No problem, hair to reopen, as with either be useful to get an explanation of the issue.

**Comment by IIAIronWolf** [comment]
> @IIAIronWolf No problem, hair to reopen, as with either be useful to get an explanation of the issue.

I agree. It hasn't been working through HACS for me, but deleting and re-adding seemed to fix it. I will update if it stops working again. Maybe a HA/HACS update broke it temporarily, and re-adding is the fix?

**Comment by Florian-Schmidt** [comment]
Unfortunately it seems to be back in 2022.4.7 🙈

Readding under _integrations_ only doesn't solve it now.

**Comment by jralph23** [comment]
Has anyone had any luck with HA 2022.5.0? I've installed HACS and downloaded the Govee repository, but the Govee integration doesn't appear when I search for it.

**Comment by Merril-r** [comment]
> Has anyone had any luck with HA 2022.5.0? I've installed HACS and downloaded the Govee repository, but the Govee integration doesn't appear when I search for it.

Did you try clearing your browser cache and and then looking for it again.

**Comment by IIAIronWolf** [comment]
This issue can be closed @LaggAt

**Comment by IIAIronWolf** [comment]
This is still needing closed.
